aspnet/migration/http-modules/sample/Asp.Net5/src/Asp.Net5/Middleware/SimpleAuthorizeMiddleware.cs

```diff
@@ -0,0 +1,35 @@
+﻿using Microsoft.AspNet.Builder;
+using Microsoft.AspNet.Http;
+using System.Threading.Tasks;
+
+namespace Asp.Net5.Middleware
+{
+    public class SimpleAuthorizeMiddleware
+    {
+        private readonly RequestDelegate _next;
+
+        public SimpleAuthorizeMiddleware(RequestDelegate next)
+        {
+            _next = next;
+        }
+
+        public async Task Invoke(HttpContext context)
+        {
+            if (context.Request.Query["iamevil"] == "true")
```

Comment:
Removed SimpleAuthorizeMiddleware because it didn't actually add to the article, and authorization is a subject on its own.